blst-native.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./lib/blst");
```

Comment:
why aren't those ts files?

Reply:
These files must be in the root so compiling them is not pretty. Took inspiration from https://github.com/connor4312/blake3

Reply:
but they ain't gonna end up in npm registry unless you modify package.json

Reply:
Done